src/main/java/fi/aalto/cs/apluscourses/presentation/module/ModuleListElementViewModel.java

```diff
@@ -35,17 +34,21 @@ public String getUrl() {
   }
 
   /**
-   * Returns the timestamp of the module if it's defined, else the URL.
+   * Returns the changelog if it's defined and an update is available,
+   * else the timestamp of the module if it's defined, else the URL.
+   *
    * @return A {@link String} with info about the module.
    */
   public String getTooltip() {
-    ZonedDateTime timestamp = getModel().getMetadata().getDownloadedAt();
-    return timestamp != null
-        ? PluginResourceBundle.getAndReplaceText(
-                "presentation.moduleTooltip.timestamp",
-                timestamp.format(DateTimeFormatter.ofLocalizedDateTime(FormatStyle.SHORT)))
-        : PluginResourceBundle.getAndReplaceText(
-                "presentation.moduleTooltip.moduleURL", getUrl());
+    var timestamp = getModel().getMetadata().getDownloadedAt();
+    var changelog = getModel().getChangelog();
+    if (isUpdateAvailable() && !changelog.equals("")) {
+      return "<html>What's new:<br>" + changelog + "</html>";
+    } else if (timestamp != null) {
+      return PluginResourceBundle.getAndReplaceText("presentation.moduleTooltip.timestamp",
```

Comment:
Nit: importing `PluginResourceBundle.getAndReplaceText` as static would make this a bit less verbose.